### PR TITLE
applications: serial_lte_modem: Address cmake warning

### PR DESCRIPTION
**Pull request by junqingzou**
Address CMake warnings for SPM and MCUBOOT subimages.
SPM to use \nrf\samples\spm\prj_minimal.conf in the future.
Currently that conf file caused SPM build error.

Signed-off-by: Jun Qing Zou <jun.qing.zou@nordicsemi.no>

### CODE REVIEW

File: applications/serial_lte_modem/child_image/mcuboot.conf [hunk 1]
```diff
@@ -6,5 +6,4 @@ CONFIG_UART_CONSOLE=n
 CONFIG_LOG_BACKEND_UART=n
 CONFIG_USE_SEGGER_RTT=y
 CONFIG_RTT_CONSOLE=y
-CONFIG_LOG_BACKEND_RTT=y
```
Comment (rlubos) [inline applications/serial_lte_modem/child_image/mcuboot.conf:9]: What was the warning this generated? Perhaps if you disable RTT log backend, other RTT configs should be removed as well?

Comment (junqingzou) [reply applications/serial_lte_modem/child_image/mcuboot.conf:9]: warning: LOG_BACKEND_RTT (defined at subsys/logging\Kconfig.backends:52) was assigned the value 'y'
but got the value 'n'. Check these unsatisfied dependencies: (!LOG_MINIMAL) (=n). See
http://docs.zephyrproject.org/latest/reference/kconfig/CONFIG_LOG_BACKEND_RTT.html and/or look up
LOG_BACKEND_RTT in the menuconfig/guiconfig interface. The Application Development Primer, Setting
Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be helpful
too.

Comment (junqingzou) [reply applications/serial_lte_modem/child_image/mcuboot.conf:9]: No such warning before, don't know from when it pops up. We don't care about MCUBOOT logging at all.

Comment (rlubos) [reply applications/serial_lte_modem/child_image/mcuboot.conf:9]: Interesting, perhaps something changed in the default mcuboot conf.

**Review by rlubos — APPROVED** [review]

**Review by lats1980 — APPROVED** [review]
LGTM.

**Pull request merged**